sw/device/lib/testing/entropy_testutils.c

```diff
@@ -50,11 +50,11 @@ void entropy_testutils_auto_mode_init(void) {
       .reseed_material =
           {
               .len = 1,
-              .data = {0x00000002},  // Reseed from entropy source only.
+              .data = {0x00000902},  // Reseed from entropy source only.
```

Comment:
i would probably recommend ORing in `kMubiFalse` into the right bit position, that way in the future if mubi values change it will be slightly more resistant.

Reply:
This is a good idea. I've changed it in both the testutils and the unittests.